### PR TITLE
DlgController: add link to xml file

### PR DESCRIPTION
**Pull request by ronso0**
In addition to the script files, also show a clickable link to the base xml file

### CODE REVIEW

File: src/controllers/dlgprefcontrollerdlg.ui [hunk 1]
```diff
@@ -318,7 +318,7 @@
              </sizepolicy>
             </property>
             <property name="text">
-             <string>Script Files:</string>
+             <string>Preset Files:</string>
```
Comment (Holzhaus) [inline src/controllers/dlgprefcontrollerdlg.ui:321]: IIRC we wanted to get rid of the "Preset" term and use "Mapping" instead.

**Review by Holzhaus — COMMENTED** [review]
Thank you!

File: src/controllers/dlgprefcontrollerdlg.ui (hunk 1 above)
Comment (ronso0) [reply src/controllers/dlgprefcontrollerdlg.ui:321]: okay, will do that for all user-visible strings.
Also I found some places were my changes can be improved.

**Comment by ronso0** [comment]
Done, ready for review!

**Review by Holzhaus — CHANGES_REQUESTED** [review]
Thanks! The "(built-in)" suffix for XML files is missing. Can you add it?

**Comment by ronso0** [comment]
> Thanks! The "(built-in)" suffix for XML files is missing. Can you add it?

Done.

**Comment by Holzhaus** [comment]
> > Thanks! The "(built-in)" suffix for XML files is missing. Can you add it?
> 
> Done.

Can you move it out of the link?

Screenshot from 2020-11-29 23-30-03

**Comment by ronso0** [comment]
> > > Thanks! The "(built-in)" suffix for XML files is missing. Can you add it?
> > 
> > 
> > Done.
> 
> Can you move it out of the link?
> 
> Screenshot from 2020-11-29 23-30-03

oh, of course.
Done

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you!

**Comment by Holzhaus** [comment]
Waiting for CI.

**Pull request merged**